data/operators/man_made/surveillance.json

```diff
@@ -923,12 +923,12 @@
       }
     },
     {
-      "displayName": "Denn's Biomarkt?",
+      "displayName": "Denns BioMarkt",
       "id": "dennsbiomarkt-9c1a6c",
       "locationSet": {"include": ["001"]},
```

Comment:
What does `001` means? Does it make sense to change it to `["at", "de"]` to be inline with the supermarket?

Reply:
These 3-digit locationSets are [UN M49 codes](https://en.wikipedia.org/wiki/UN_M49). In this case, `001` would include the entire world. If this operator is only present in Austria and Germany, then it should be changed. You can use [this website](https://location-conflation.com/?referrer=nsi) to help visualize locationSets.

Reply:
thanks! updated.